### ISSUE TITLE
Policy for syntax licenses

### ISSUE THREAD

**Issue by reknih** [sharkdp/bat#2566]
It would be great if the README or the [assets documentation](https://github.com/sharkdp/bat/blob/master/doc/assets.md) would mention the licenses eligible for inclusion in `bat`.

With a very cursory search, I was not able to find licenses other than MIT or Apache 2.0 for syntaxes. Is this a hard criterion? If not, it would be great to have a list of syntaxes with alternative licenses.

**Motivation:** I would like to use `bat`'s set of syntaxes as a syntect dump in another project and comply with the attribution and copyleft requirement any syntax licenses may have.

**Comment by keith-hall** [comment]
I believe we accept any license which is compatible with `bat`s license:
https://github.com/sharkdp/bat/pull/1960

Currently we only automatically reject GPL assets as I can see:
https://github.com/sharkdp/bat/pull/1961/files#diff-1bbacd0148e7605b127a765f5f0e6c04b84d3c471bac8d2e612a01d9b00cf39c

**Comment by sharkdp** [comment]
Exactly. Another thing that might be useful is to call `bat --acknowledgements`, which shows licenses for all syntaxes (and themes) that are included.
